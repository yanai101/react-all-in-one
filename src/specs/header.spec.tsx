import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Header from '../components/heder'; 

describe('<Header/>', () => {
    test('Should render corrently', () => {
        const component = renderer.create(<Header/>);
        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});