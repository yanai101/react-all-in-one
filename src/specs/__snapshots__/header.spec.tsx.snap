// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> Should render corrently 1`] = `
<div
  className="headerCon"
>
  <header
    className="headerFont"
  >
    hover on the up title  !!!
  </header>
</div>
`;
